apps/.eslintrc.js

```diff
@@ -101,6 +101,7 @@ module.exports = {
     Turtle: 'readonly',
     YT: 'readonly',
   },
+  reportUnusedDisableDirectives: true,
```

Comment:
`true` means `'warn'` here. the docs say how to specify `'error'`: https://eslint.org/docs/latest/use/configure/rules#report-unused-eslint-disable-comments but this does not seem to be possible for us, perhaps because we are still using .eslintrc.js rather than eslint.config.js.